plotly/tests/test_core/test_jupyter/test_jupyter.py

```diff
@@ -0,0 +1,46 @@
+"""
+test__jupyter
+
+"""
+from __future__ import absolute_import
+
+import nbformat
+from nbconvert import HTMLExporter
+from nbconvert.preprocessors import ExecutePreprocessor
+
+from unittest import TestCase
+from os import path
+import subprocess
+
+import plotly
+
+PATH_ROOT = path.dirname(__file__)
+PATH_FIXTURES = path.join(PATH_ROOT, 'fixtures')
+PATH_TEST_NB = path.join(PATH_FIXTURES, 'test.ipynb')
+PATH_TEST_HTML = path.join(PATH_FIXTURES, 'test.html')
+
+
+class PlotlyJupyterTestCase(TestCase):
+    def setUp(self):
```

Comment:
Looks like `setUp` is what we want here

![image](https://cloud.githubusercontent.com/assets/6675409/17784450/8bcb43fa-654a-11e6-932a-05df7e190231.png)

but I'm no python expert.